### PR TITLE
fix(android): hide surfaceView for loading time when shutter is hidden

### LINKED ISSUE #4045: [BUG]: Black screen on Android until video fully loaded
### Version

6.4.3

### What platforms are you having the problem on?

Android

### System Version

android 12+

### On what device are you experiencing the issue?

Real device, Simulator

### Architecture

Old architecture

### What happened?

I have a static photo and wanted to add a "live photo". This is a regular mp4 video that is uploaded via a link and activated.

But the problem is that in the latest versions of the library at the moment of "onLoadStart" a black canvas is called, which is shown until the video is loaded. I saw shutterColor and hideShutterView in the documentation, but using them doesn't solve the problem.

### Reproduction

_No response_

### Reproduction

Step to reproduce this bug are: 
Just try to loaded video from remote uri: for example https://d23dyxeqlo5psv.cloudfront.net/big_buck_bunny.mp4

### PR DESCRIPTION
**Pull request by moskalakamil**
## Summary
Fix black screen when `hideShutterView` is true
Closes #4045
### Motivation
Fix #4045, #4006, #2974, #2483
### Changes
- Hide surface view for loading time, when shutter view is hidden
- Allow to enter 'transparent' in `shutterColor` prop
## Test plan
run example app on android with Video `hideShutterView={true}` prop - there is no black screen then

### CODE REVIEW

File: android/src/main/java/com/brentvatne/exoplayer/ExoPlayerView.java [hunk 1]
```diff
@@ -161,8 +161,22 @@ public void updateSurfaceView(@ViewType.ViewType int viewType) {
         }
     }
 
+    private void hideShutterView() {
+        shutterView.setVisibility(INVISIBLE);
+        ((SurfaceView)surfaceView).setAlpha(1);
```
Comment (freeboub) [inline android/src/main/java/com/brentvatne/exoplayer/ExoPlayerView.java]: What about when the user configures the texture view ?

**Review by freeboub — CHANGES_REQUESTED** [review]

**Review by bradfx — APPROVED** [review]

**Review by KrzysztofMoch — APPROVED** [review]

File: android/src/main/java/com/brentvatne/exoplayer/ExoPlayerView.java (hunk 1 above)
Comment (KrzysztofMoch) [reply android/src/main/java/com/brentvatne/exoplayer/ExoPlayerView.java]: this was cast only - it's removed so it should be fine now

**Review by freeboub — APPROVED** [review]

**Comment by freeboub** [comment]
I approve because code is OK, but I am not sure this is the expected behavior.
In attached ticket, peoples are asking to get previous video image when they load a new video ... here it will hide the previous video (opacity 0) !

**Comment by CMLCNL** [comment]
When you can release this version? thank you.

**Pull request merged**